### PR TITLE
be more explicit on egg dependencies

### PR DESCRIPTION
**Pull request by tomgross**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 )
 
 
-version = '0.3'
+version = '0.3.post3.dev0'
```
Comment (rnixx) [inline setup.py]: 0.4.dev0

File: setup.py [hunk 2]
```diff
@@ -18,27 +18,35 @@
       long_description=longdesc,
       classifiers=[
           'Environment :: Web Environment',
+          "Framework :: Plone",
+          "Framework :: Plone :: 5.0",
```
Comment (rnixx) [inline setup.py]: This package also works with plone 4.3 and maybe earlier versions

File: setup.py [hunk 3]
```diff
@@ -18,27 +18,35 @@
...
-          'Plone',
+          'plone.behavior',
           'plone.dexterity',
+          'zope.annotation',
+          'zope.component',
+          'zope.interface',
       ],
       extras_require={
           'test': [
+              'plone.app.event',
```
Comment (rnixx) [inline setup.py]: why plone.app.event as test dependency?

File: src/collective/instancebehavior/tests/test_instancebehavior.py [hunk 4]
```diff
@@ -1,4 +1,5 @@
-import unittest2 as unittest
+# -*- coding: utf-8 -*-
```
Comment (rnixx) [inline src/collective/instancebehavior/tests/test_instancebehavior.py]: Not that it concerns me, but coding headers make only sense in files containing non ASCII chars. Seen this quite often recently that people add this headers without any need...

Comment (tomgross) [inline src/collective/instancebehavior/tests/test_instancebehavior.py]: code-analysis complains, if the header is not there

File: setup.py (hunk 3 above)
Comment (tomgross) [inline setup.py]: It is imported in the doctests in README.rst and __init__.py

File: src/collective/instancebehavior/tests/test_instancebehavior.py (hunk 4 above)
Comment (gforcada) [inline src/collective/instancebehavior/tests/test_instancebehavior.py]: @rnixx fair point, the question is if people will remember to add it as soon as they add a non-ASCII character... so mostly a safety belt

Comment (rnixx) [inline src/collective/instancebehavior/tests/test_instancebehavior.py]: @gforcada if you add non ASCII characters to source without setting this header you'll get a syntax error. Anyway, nevermind ;)

File: setup.py (hunk 3 above)
Comment (rnixx) [inline setup.py]: Ah, right. Maybe it's worth to use another behavior for testing then.

**Comment by tomgross** [comment]
@rnixx Thanks for reviewing. Any chance to get this merged and released?

**Comment by rnixx** [comment]
@tomgross sure

**Comment by rnixx** [comment]
https://pypi.python.org/pypi/collective.instancebehavior

**Comment by tomgross** [comment]
@rnixx Thanks!

**Pull request merged**